website/react-magma-docs/src/pages/api/dropdown.mdx

```diff
@@ -85,6 +85,66 @@ export function Example() {
 }
 ```
 
+## Expandable Menu Items
```

Comment:
I wonder if @orion-cengage has an opinion on where this should go as far as the order. I wonder if it should go under `DropdownMenuNavItem`
![image](https://github.com/cengage/react-magma/assets/91160746/7972bcc5-aedd-428a-91d7-f679a62e0a18)

Reply:
Orion agreed, moved section under `DropdownMenuNavItem`.